website/src/documentation/how-tos.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: section
+title: "How Tos"
+section_menu: section-menu/documentation.html
+permalink: /documentation/how-tos/
+redirect_from:
+  - /learn/how-tos/
+  - /docs/learn/how-tos/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# How Tos
+
+Welcome to the [Apache Beam](https://beam.apache.org/) How Tos page. This page contains a collection of resources that will help you to get started and master Apache Beam. If you're just starting, you can view this as a guided tour, otherwise you can jump straight to any section of your interest.
```

Comment:
perhaps "help you to get started and use Apache Beam"

Reply:
Changed